### ISSUE TITLE
==[ Disconnected from Server ]==   (Waterdog)

### ISSUE THREAD

**Issue by WolfyRed** [yesdog/Waterdog#178]
**Version of Waterdog**
git:Waterdog-Bootstrap:1.15-SNAPSHOT:f1796c0:316
*(Latest version according to [the download page](https://ci.codeme.org/job/yesdog/job/Waterdog/))*

**Expected Behavior**
To join and for the server to let me in, with no/barely any issues.

**Actual Behavior**
When I join, I cannot get into the server. I only get kicked out in the process of joining.

**Crash logs?**
There has been no crash logs, it's a Bedrock client problem only.

**Describe the bug**
I join the server, it kicks me out immediately. It's just a matter of milliseconds. 

**To Reproduce**
1. Get the latest version of Waterdog *(Supposedly from the [official download page](https://ci.codeme.org/job/yesdog/job/Waterdog/))*
2. Start up the Waterdog proxy
3. Join it from Minecraft Bedrock!
4. Get kicked immediately

**Screenshots**
![image](https://user-images.githubusercontent.com/66114400/108167777-36c96000-70c4-11eb-9df7-31907da91737.png)

**Server software used:**
CraftBukkit version git-Spigot-dcd1643-e60fc34 / Geyser version 1.2.0-SNAPSHOT *(Bedrock: 1.16.100/1.16.101 - 1.16.200/1.16.201)*

**Additional context**
*I can connect with the new WaterdogPE Proxy, but that proxy does not and/or cannot use Bungee/Velocity plugins.*

**Comment by Alemiz112** [comment]
Hi, please use build from our new [Jenkins server](https://jenkins.waterdog.dev/job/Waterdog/job/Waterdog/job/master-zlib/lastSuccessfulBuild/artifact/Waterfall-Proxy/bootstrap/target/Waterdog.jar).

**Comment by WolfyRed** [comment]
> Hi, please use build from our new [Jenkins server](https://jenkins.waterdog.dev/job/Waterdog/job/Waterdog/job/master-zlib/lastSuccessfulBuild/artifact/Waterfall-Proxy/bootstrap/target/Waterdog.jar).

Yes that one doesn't kick me off immediately, it just kicks me off half of the time and the other half of the time it loads forever
